sdk/include/microvium/microvium_port.h

```diff
@@ -165,13 +166,13 @@ fixes and improvement from the original github or npm repository.
  *
  * Offset may be negative
  */
-#define MVM_LONG_PTR_ADD(p, s) ((MVM_LONG_PTR_TYPE)((uint8_t*)p + (intptr_t)s))
+#define MVM_LONG_PTR_ADD(p, s) ((MVM_LONG_PTR_TYPE)((uint8_t *)p + (intptr_t)s))
```

Comment:
Maybe, while you're here,
```suggestion
#define MVM_LONG_PTR_ADD(p, s) ((MVM_LONG_PTR_TYPE)((uint8_t *)p + (ptrdiff_t)s))
```